Examples/Framework/include/ActsExamples/EventData/TrackJet.hpp

```diff
@@ -0,0 +1,99 @@
+// This file is part of the ACTS project.
+//
+// Copyright (C) 2016 CERN for the benefit of the ACTS project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include "Acts/Definitions/Algebra.hpp"
+
+#include <iostream>
+#include <vector>
+
+namespace ActsExamples {
+// TODO: Capitalize enum names for consistency
+enum jetlabel { unknown = -99, ljet = 0, cjet = 4, bjet = 5 };
+// TODO: Remove hadronlabel
+enum hadronlabel {
```

Comment:
Where these the final enums that we were using? I thought I remember they were moved to a central place and made consistent. Can you check?

If the jet class lives in Core, these enums also have to live in Core to be usable.